tests/NHSD.BuyingCatalogue.Organisations.Api.IntegrationTests/Steps/OrganisationsSteps.cs

```diff
@@ -242,5 +344,12 @@ private sealed class OrganisationTable
 
             public string Country { get; init; }
         }
+
+        private sealed class RelatedOrganisationTable
```

Comment:
Let's add the `[UsedImplicitly(ImplicitUseTargetFlags.Members)]` attribute to avoid false positives in R#.